rxlifecycle/src/main/java/com/trello/rxlifecycle/RxLifecycle.java

```diff
@@ -135,6 +140,57 @@ public Boolean call(R lifecycleEvent) {
         return bind(lifecycle, FRAGMENT_LIFECYCLE);
     }
 
+    /**
+     * Binds the given source a View lifecycle.
+     * <p>
+     * Use with {@link Observable#compose(Observable.Transformer)}:
+     * {@code source.compose(RxLifecycle.bindView(lifecycle)).subscribe()}
+     * <p>
+     * This helper automatically determines (based on the lifecycle sequence itself) when the source
+     * should stop emitting items. For views, this effectively means watching for a detach event and
+     * unsubscribing the sequence when one occurs.
+     * <p>
+     * Note that this will unsubscribe after the first {@link ViewAttachEvent.Kind#DETACH} event is received,
+     * and will not resume if the view is re-attached later.
+     *
+     * @param view the view to bind the source sequence to
+     * @return a reusable {@link Observable.Transformer} that unsubscribes the source during the View lifecycle
+     */
+    public static <T> Observable.Transformer<T, T> bindView(final View view) {
+        if (view == null) {
+            throw new IllegalArgumentException("View must be given");
+        }
+
+        return bindView(RxView.detaches(view));
+    }
+
+    /**
+     * Binds the given source a View lifecycle.
+     * <p>
+     * Use with {@link Observable#compose(Observable.Transformer)}:
+     * {@code source.compose(RxLifecycle.bindView(lifecycle)).subscribe()}
+     * <p>
+     * This helper automatically determines (based on the lifecycle sequence itself) when the source
+     * should stop emitting items. For views, this effectively means watching for a detach event and
+     * unsubscribing the sequence when one occurs. Note that this assumes <em>any</em> event
+     * emitted by the given lifecycle indicates a detach event.
+     *
+     * @param lifecycle the lifecycle sequence of a View
+     * @return a reusable {@link Observable.Transformer} that unsubscribes the source during the View lifecycle
+     */
+    public static <T, E> Observable.Transformer<T, T> bindView(final Observable<E> lifecycle) {
```

Comment:
Mimic [Observable#takeUntil](https://github.com/ReactiveX/RxJava/blob/1.x/src/main/java/rx/Observable.java#L8231-L8233) and use `<? extends E>`.

Btw, this is the difference between the three: 
- `Observable<E>` an Observable that only outputs E and exactly that.
- `Observable<? extends E>` an Observable that outputs anything with an upperbound of E (so any descendant of E or E itself).
- `Observable<?>` an Observable that outputs anything, can't use that here because the Operator (Transformer) expects a type!